### PR TITLE
feat: added recipe to send message to contact

### PR DESCRIPTION
**Pull request by mneri-whispir**
Added sample code for creating a contact and sending a simple message

### CODE REVIEW

File: examples/lib/contact.lib.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+const { ContactsApi } = require('@whispir/ztest-node-api')
```
Comment (richardfontein) [inline examples/lib/contact.lib.js:1]: Should this package instead be `@whispir/whispir-node`?

File: examples/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@whispir/ztest-node-api": "^1.0.0",
```
Comment (richardfontein) [inline examples/package.json:13]: As above, should this be `@whispir/whispir-node`?

File: examples/package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@whispir-node/recipes",
```
Comment (richardfontein) [inline examples/package.json:2]: To keep consistent with the `@whispir` scope, should we instead rename this to `@whispir/node-recipes`?

File: examples/whispir-send-message-to-contact/index.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+//     }
+
+//     const result = await contactApi.getContactsById(
+//         WORKSPACE_ID,
+//         API_KEY,
+//         'B5589BB86E0DD79D',
+//         "application/vnd.whispir.contact-v1+json", null, null, null, requestConfig)
+
+//     return result;
+// }
```
Comment (richardfontein) [inline examples/whispir-send-message-to-contact/index.js:35]: Got some commented code here, I think it can be removed.

**Review by richardfontein — CHANGES_REQUESTED** [review]

**Comment by mneri-whispir** [comment]
@richardfontein updated with the needed changes. kindly review again. thank you!

**Pull request closed without merging**